src/main/java/duke/Task/Task.java

```diff
@@ -0,0 +1,38 @@
+package duke.Task;
+
+public abstract class Task {
+    private final String taskDescription;
+    private boolean isDone;
+
+    protected Task(String task) {
+        this.taskDescription = task;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns the description of task
+     * @return string representation of the description of task
+     */
+    public String getTaskDescription() {
+        return this.taskDescription;
+    }
+
+    /**
+     * Returns true if object is completed, false otherwise
+     * @return true if object is completed, false otherwise
+     */
+    public boolean isCompleted() {
```

Comment:
"isDone" and "isCompleted" seems kind of similar... Would it be better if you change "isCompleted" to "getCompletionStatus"?

Reply:
Perhaps changing it to get would be a good idea as "isXxx" sounds like a boolean variable